### PR TITLE
Winston ES Logging Enhancement

### PR DESCRIPTION
**Pull request by reynldi**
### What does this MR do?
Winston Elasticsearch Logging Enhancement

- Adding default log format
- Improving Http log Format
- Improving JSON masking

### Author's Checklist
- [x] Follow the Documentation Guidelines
- [x] Use label properly
- [x] Assign to Reviewer

### New Dependency
_put dependency link_

### CODE REVIEW

File: src/app/middleware/apm-agent.middleware.ts [hunk 1]
```diff
@@ -9,11 +9,17 @@ class APMAgent {
...
     captureHeaders: true,
+    captureBody: 'all',
+    usePathAsTransactionName: true,
   });
 
   public init() {
     return this._init.middleware.connect();
   }
+
+  public captureError(error) {
```
Comment (tahsinature) [inline src/app/middleware/apm-agent.middleware.ts]: I think no need to implement to capture method, it automatically logs the error
even tho if we dont log them

File: src/app/util/mask-json/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const collection = require('./collection-key.json');
+import { cloneDeepWith, isObject, toLower, some } from 'lodash';
+
+/**
+ * Filtering and Censoring Sensitive Data
+ * All data keys are stored in data-key.json
+ */
+
+const replacement = '******';
+const maskJson = (values): any => {
```
Comment (tahsinature) [inline src/app/util/mask-json/index.ts:10]: can cleanup like this
```js
const maskJson = (values): any =>
  cloneDeepWith(values, (value, key: string) => {
    if (some(collection, (item) => key === item)) return replacement;
    if (!isObject(value)) return value;
  });```

**Review by tahsinature — CHANGES_REQUESTED** [review]
please review the `requested changes`

File: src/app/middleware/apm-agent.middleware.ts (hunk 1 above)
Comment (reynldi) [reply src/app/middleware/apm-agent.middleware.ts]: still figure out. It's not captured

**Pull request merged**